app/src/main/java/ru/chaichuk/hwapp/MoviesLIstAdapter.kt

```diff
@@ -0,0 +1,110 @@
+package ru.chaichuk.hwapp;
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.RatingBar
+import android.widget.TextView
+import androidx.core.content.res.ResourcesCompat
+import androidx.recyclerview.widget.RecyclerView
+import ru.chaichuk.hwapp.data.models.Movie
+
+
+class MoviesListAdapter(private val clickListener: OnRecyclerItemClicked) :
+    RecyclerView.Adapter<MoviesListViewHolder>() {
+
+    private var movies = listOf<Movie>()
+
+    override fun getItemViewType(position: Int): Int {
+        return when (movies.size) {
+            0 -> VIEW_TYPE_EMPTY
```

Comment:
here also the same situation 
why need not used viewType ?

Reply:
removed